client/stylesheets/_toCat.scss

```diff
@@ -16,7 +16,7 @@ GLOBAL | GLOBAL DIVS | HEADER | FOOTER **/
 }
 
 .tooltip > .tooltip-inner {
-    background-color: $color-spacered !important;
+    background-color: $color-spacegrey !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces